packages/synapse-react-client/test/lib/containers/PageProgress.test.tsx

```diff
@@ -5,72 +5,74 @@ import PageProgress, {
   const props: PageProgressProps = {
     barColor: '',
     barPercent: 30,
-    backBtnLabel: 'Go back',
+    backBtnLabel: buttonLabels.back,
     backBtnCallback: onBackButtonClicked,
-    forwardBtnLabel: 'Go forward',
+    forwardBtnLabel: buttonLabels.forward,
     forwardBtnCallback: onNextButtonClicked,
     forwardBtnActive: false,
   }
   const canGoNextProps: PageProgressProps = { ...props }
   canGoNextProps.forwardBtnActive = true
 
   it('render component without crashing', () => {
-    const { container } = render(<PageProgress {...props} />)
-    expect(container).toBeDefined()
+    const { component } = setUp(props)
+    expect(component).toBeDefined()
   })
 
-  it('should render progress bar with correct width', () => {
-    const { container } = render(<PageProgress {...props} />)
-    expect(
-      container.querySelector('.page-progress-percent')?.getAttribute('style'),
-    ).toBe('width: 30%;')
+  it('should render progress bar with correct percentage', () => {
+    const { progressbar } = setUp(props)
+    expect(progressbar).toHaveAttribute('aria-valuenow', '30')
   })
 
-  it('should render button with correct labels', () => {
-    render(<PageProgress {...props} />)
+  it('should render buttons with correct labels', () => {
+    const { buttons } = setUp(props)
+    expect(buttons.back).toBeInTheDocument()
+    expect(buttons.forward).toBeInTheDocument()
+  })
 
-    expect(
-      screen
-        .getByRole('button', { name: 'Go back' })
-        .classList.contains('btn-progress-back'),
-    ).toBe(true)
-    expect(
-      screen
-        .getByRole('button', { name: 'Go forward' })
-        .classList.contains('btn-progress-next'),
-    ).toBe(true)
+  it('should disable forward button when cannot advance', () => {
+    const { buttons } = setUp(props)
+    expect(buttons.forward).toBeDisabled()
   })
 
-  it('should have correct css class for forward button', () => {
-    render(<PageProgress {...canGoNextProps} />)
-    expect(
-      screen
-        .getByRole('button', { name: 'Go forward' })
-        .classList.contains('btn-progress-next-active'),
-    ).toBe(true)
+  it('should not disable forward button when can advance', () => {
+    const { buttons } = setUp(canGoNextProps)
+    expect(buttons.forward).not.toBeDisabled()
   })
 
   it('should call back button callback when clicked', async () => {
-    render(<PageProgress {...props} />)
-    await userEvent.click(screen.getByRole('button', { name: 'Go back' }))
+    const { user, buttons } = setUp(props)
+    expect(buttons.back).not.toBeDisabled()
+    await user.click(buttons.back)
     expect(onBackButtonClicked).toHaveBeenCalledTimes(1)
   })
 
-  it('should not call forward button callback when it is not active', async () => {
-    render(<PageProgress {...props} />)
-    await userEvent.click(screen.getByRole('button', { name: 'Go forward' }))
-    expect(onNextButtonClicked).toHaveBeenCalledTimes(0)
-  })
-
```

Comment:
Test no longer relevant -- the button is shown to be disabled when not active in another test. A disabled button can't be clicked, so can't confirm that the callback isn't called